src/compute-model-statistics/src/main/scala/ComputeModelStatistics.scala

```diff
@@ -113,11 +109,50 @@ class ComputeModelStatistics(override val uid: String) extends Transformer with
   /** @group setParam */
   def setEvaluationMetric(value: String): this.type = set(evaluationMetric, value)
 
-  lazy val logger = Logger.getLogger(this.getClass.getName)
+  // Parameter required only is using native Spark ML Estimator instead of MMLSpark Estimator
+  val modelName: Param[String] =
+    StringParam(this, "modelName", "Model name, only required if using SparkML estimators", "SparkML")
+
+  def getModelName: String = $(modelName)
+
+  def setModelName(value: String): this.type = set(modelName, value)
+
+  /** Parameter required only is using native Spark ML Estimator instead of MMLSpark Estimator
+    * Set to true if computing metrics for the output of a classification model, set to false if regression model.
+    */
+  val classifier: Param[Boolean] =
+    BooleanParam(this, "classifier", "Is classifier scored data, only required if using SparkML estimators", true)
+
+  def getClassifier: Boolean = $(classifier)
+
+  def setClassifier(value: Boolean): this.type = set(classifier, value)
+
+  val scoredLabelsCol =
+    StringParam(this, "scoredLabelsCol", "Scored labels column name, only required if using SparkML estimators")
```

Comment:
Could you elaborate on the differences between scores and scoredLabels? I'm having a hard time understanding why these need to both be specified

Reply:
This is applicable in classification.  The score is the raw value or probability.  The scored label is the "label" that is assigned based on the score, eg if you had a probability of 0.9 to determine whether the image is of a dog (1) or cat (0), the label would be "dog"